spec/features/admin/budget_groups_spec.rb

```diff
@@ -49,6 +49,8 @@
     scenario "Defaults to 1 heading per group", :js do
       visit admin_budget_path(group.budget)
 
+      expect(page).to have_content('Maxium number of headings in which a user can vote 1 of 3')
```

Comment:
That "Maxium number" looks like a typo...can you fix it please @iagirre? Also there's another one in line 71.